autogpt_platform/README.md

```diff
@@ -28,9 +28,9 @@ To run the AutoGPT Platform, follow these steps:
 
 3. Run the following command:
    ```
-   cp supabase/docker/.env.example .env
+   cp supabase/docker/.env.example .env.local
    ```
-   This command will copy the `.env.example` file to `.env` in the `supabase/docker` directory. You can modify the `.env` file to add your own environment variables.
+   This command will copy the `.env.example` file to `.env.local` in the `supabase/docker` directory. You can modify the `.env.local` file to add your own environment variables.
```

Comment:
I think this is unrelated to the documentation described on the PR, can we revert this?

Reply:
I initially targeted this to the master branch and GH changed the target branch to dev. I just rebased the branch to dev.